src/core/services/meta/meta.spec.js

```diff
@@ -0,0 +1,95 @@
+describe('$$mdMeta', function() {
+
+  beforeEach(module('material.core'));
+
+  describe('set meta', function () {
+    beforeEach(function () {
+      angular.element(document.getElementsByTagName('meta')).remove();
+    });
+
+    it('should create the element and append it to the body', inject(function($$mdMeta) {
```

Comment:
Does it make sense to just inject in a beforeEach instead of within every test (since it's used everywhere)? I've always hated the inject per test design.

Something like this (with or without the extra method).
```
var injectLocals = function($$mdMeta) {
  mdMeta = $$mdMeta;
};

beforeEach(function() {
  inject(injectLocals);
});
 ```